packages/pds/src/db/migrations/20230504T210229992Z-block-third-party.ts

```diff
@@ -0,0 +1,112 @@
+import { Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .alterTable('post')
+    .addColumn('replyBlocked', 'int2', (col) => col.defaultTo(0).notNull())
+    .execute()
+  await db.schema
+    .alterTable('post_embed_record')
+    .addColumn('embedBlocked', 'int2', (col) => col.defaultTo(0).notNull())
+    .execute()
+}
+
+export async function down(db: Kysely<unknown>): Promise<void> {
+  await db.schema.alterTable('post').dropColumn('replyBlocked').execute()
+  await db.schema
+    .alterTable('post_embed_record')
+    .dropColumn('embedBlocked')
+    .execute()
+}
+
+export function dataMigrationQbs(db: Kysely<Schema>) {
```

Comment:
where does this get run? is it an app migration?

Reply:
It is effectively an app migration, but it is simple enough to be run manually.  So I just documented it here for good measure.  I can also just wrap it into an app migration, maybe that's a good idea.

Reply:
Ahh I started pulling these out into an app migration but the more I think about it, I think it's best they just be run manually.  I am going to add a comment here to make it clearer what's going on.